public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+	"roadmap.hard_fork.3": "An initial release of the packages repository will be created and migration from `nix` to `aux` naming will begin to reduce confusion and errors now that the two projects have diverged.",
+	"roadmap.organization.title": "Strukturierung",
+	"roadmap.organization.1": "The packages repository will have sets extracted to allow for Special Interest Groups to more easily manage their lifecycles. For example, `pythonPackages` will be moved to a separate repository that will be the reponsibility of a Python SIG if there is interest.",
+	"roadmap.organization.2": "Flakes will be standardized with its current implementation as a v0. While not ideal, the feature is used far too widely to be changed or removed without breaking the ecosystem. Instead, this v0 implementation will be enabled and future work for Flakes that addresses its shortcomings may be handled by a Flakes SIG.",
+	"roadmap.organization.3": "The `aux` CLI will be modified to provide more ergonomic management of packages and systems. Additional subcommands such as `aux system switch` and `aux system build` will be added to make onboarding and ongoing maintenance easier.",
+	"roadmap.infra.title": "Infrastruktur",
+	"roadmap.infra.1": "Now that the project has significantly diverged from upstream, we will need to provide our own Continuous Integration and Binary Cache services. Existing governance structures will be used to manage the adoption of these technologies.",
+	"roadmap.infra.2": "Documentation will be included as a part of the Continuous Integration process and will have Continuous Deployment configured to keep it up to date, making it easier for contributors to create educational materials.",
+	"roadmap.infra.3": "Search infrastructure for packages and options will also be created.",
+	"roadmap.alignment.title": "Un­ab­hän­gig­keit",
+	"roadmap.alignment.1": "By this point, we've done it. We have created a sustainable, independent fork of the Nix ecosystem. Now that the initial work has been completed we will begin the governance rotation process with our first elections and education as to how project governance works.",
+	"roadmap.alignment.2": "Documentation, project branding, and packages will be polished and presented as a fully stable release. From this point forward the Steering Committee will be managing the creation of Special Interest Groups and other necessary Committees. All enhancements will be suggested in Aux Enhancement Proposals which encourage existing SIGs to implement features or encourage the formation of a new SIG to be created for such a purpose.",
+
+	"values.title": "Die Werte",
+	"values.rep.title": "Vertretung",
+	"values.rep": "Die Ansichten der Mitwirkenden sollten vertreten werden und die Organisationsstruktur sollte demokratisch ausgerichtet sein.",
+	"values.kindness.title": "Wohlwollen",
+	"values.kindness": "Das Projekt sollte einladend sein und einen sicherer Ort für die Teilnahme darstellen.",
```

Comment:
Maybe 'Mitwirkung' instead of 'Teilnahme'?

Reply:
Could also be rephrased a bit more like "Das Projekt soll zum Mitwirken einladen und eine sichere Umgebung für Anteilnahme sein."?

German is a fun language, for sure :^)

Reply:
Yeah it is...

@christoph-heiss I like your rewrite, thanks.

Except I'm not sure about 'Anteilnahme', it sounds weirdish to me given the connotation of 'grief'. 

What about 'Beteiligung' instead?

Reply:
Changed to 'Beteiligung'